libbeat/docs/shared-env-vars.asciidoc

```diff
@@ -49,3 +65,38 @@ and what each configuration looks like after replacement:
 |`name: ${NAME:beats}` |no setting            |`name: beats`
 |`name: ${NAME:beats}` |`export NAME=elastic` |`name: elastic`
 |==================================
```

Comment:
oh, I did forget `go-ucfg` always appending the full settings name and source when reporting an error.

I'm ok with not documenting the `:+` operator, as it's somewhat complex. About the error operator `:?` I'm somewhat on the fence. I'm ok with just leaving it out or maybe adjusting the error message in go-ucfg itself.